ecosystem/platform/server/config/routes.rb

```diff
@@ -20,8 +20,14 @@
     resources :users, only: %i[show update]
   end
 
+  # KYC routes
+  get 'onboarding/kyc_redirect', to: 'onboarding#kyc_redirect'
+  get 'onboarding/kyc_callback', to: 'onboarding#kyc_callback'
+
   get 'onboarding/email', to: 'onboarding#email'
-  post 'onboarding/email', to: 'onboarding#email_update'
+  post 'onboarding/email', to: 'onboarding#kyc_callback'
```

Comment:
Can you explain this change? I don't understand at first glance.

Reply:
typo, fixed!